docs/tutorial/workers.md

```diff
@@ -17,13 +17,13 @@ search:
 
 ## Why workers and work pools?
 
-Workers and work pools bridge the Prefect orchestration layer with the infrastructure the flows are actually executed on. 
+Workers and work pools are a bridge between the Prefect orchestration layer and infrastructure that enable flow runs to be executed in dynamically provisioned environments. To transition from persistent infrastructure to dynamic infrastructure, you can use `flow.deploy` instead of `flow.serve`.
 
-!!! tip "[Choosing Between workers and `flow.serve()`](/concepts/deployments/#two-approaches-to-deployments)"
+!!! tip "[Choosing Between `flow.deploy()` and `flow.serve()`](/concepts/deployments/#two-approaches-to-deployments)"
```

Comment:
My comment is regarding an unedited section on line 23:

"Just remember, if infrastructure needs escalate" should be
"Just remember, if infrastructure needs to escalate"

Reply:
I think it's correct as written. I think if we were to describe the infrastructure rather than the needs we'd say something like, "Just remember, if infrastructure needs to scale up..."